Handheld/Handheld.pro

```diff
@@ -24,13 +24,16 @@ ARCGIS_RUNTIME_VERSION = 200.5.0
 DEFINES += ARCGIS_MAPS_SDK_VERSION=$$ARCGIS_RUNTIME_VERSION
 include($$PWD/../Shared/build/arcgisruntime.pri)
 
+include($$PWD/../3rdparty/zlib_minizip_ng.pri)
```

Comment:
Except if I miss something, the path is not correct. The zlib files are in `3rdparty`